cmd/ebitenmobile/gomobile.go

```diff
@@ -169,6 +178,19 @@ import (
 		return tmp, err
 	}
 
+	if err := os.Mkdir(filepath.Join("src", "_files"), 0755); err != nil {
+		return tmp, err
+	}
+	if err := os.WriteFile(filepath.Join("src", "_files", "EbitenViewController.m"), objcM, 0644); err != nil {
+		return tmp, err
+	}
+	if err := os.WriteFile(filepath.Join("src", "_files", "EbitenView.java"), viewJava, 0644); err != nil {
+		return tmp, err
+	}
+	if err := os.WriteFile(filepath.Join("src", "_files", "EbitenSurfaceView.java"), surfaceViewJava, 0644); err != nil {
+		return tmp, err
+	}
```

Comment:
I don't understand why the files are written here.

Reply:
Because we are writing `gobind.go` to tmp folder, it has `//go:embed` and that's why we need to have those files available

Reply:
Embedding is done only at `go build` and we are building it on user's machine that doesn't have those files

Reply:
Ah, I see...